decidim/decidim
# Allow to publish an Election even if it hasn't valid Questions

#### :tophat: What? Why?

When creating a new Election, it's not possible to publish it, and there isn't any message on why it isn't possible. That's because there isn't any questions/answers published yet. 

I actually forget about that and didn't know why that was happening, so after talking with @carolromero we agreed that this should work the same as other resources, without this kind of special logic.

Mind that even though it's possible to publish it, the final sanity check is done while setting up the Election (aka sending it to the Bulletin Board):

https://github.com/decidim/decidim/blob/fd0e09e9e28ab19c78fce193a2bcf4fdb0c79811/decidim-elections/app/forms/decidim/elections/admin/setup_form.rb#L41

#### Testing

1. Sign in as admin
2. Go to admin dashboard
3. Create an Election
4. See that you now can publish it

### :camera: Screenshots


#### Before 

![Screenshot of elections' management before the fix](https://github.com/decidim/decidim/assets/717367/93568a35-12cf-440f-b555-96bd89089021)

#### After

![Screenshot of elections' management after the fix](https://github.com/decidim/decidim/assets/717367/b021ea53-8606-4807-afc9-d7c81ad484ba)


:hearts: Thank you!
